.travis.yml

```diff
@@ -35,6 +35,11 @@ notifications:
       - "chat.freenode.net#oppia"
     on_success: never
     on_failure: always
+  webhooks:
+    urls:
+      - https://webhooks.gitter.im/e/f8f782497ec1ffcea2e7
```

Comment:
Ah, I see, thanks for explaining!

Maybe add a comment like: "This URL can be obtained by going to the Gitter chat room and clicking Settings > Integrations." Then I think this can be merged.